styles.css

```diff
@@ -119,3 +123,24 @@ p {
   transition: all 0.3s ease 0s;
   transform: scale(1.1);
 }
+
+.card-deck {
+  margin: 35px;
+  margin-top: 25px;
+}
+
+.card {
+  min-width: 16%;
+  max-width: 20%;
+  margin-top: 30px;
+  border-color: transparent;
+}
+
+.card-body-last {
+  padding-top: 200px;
+  padding-left: 10%;
+}
+
+.fa-long-arrow-alt-right {
+  margin-left: 20px;
```

Comment:
Hiya! Sebastian here. Your intention here is to add a little bit of space between the icon and the "See all" text, isn't it?
While your solution works, it's not ideal, because you'd change every `fa-long-arrow-alt-right` with it, regardless of where you use them. In your example there is only one, but this might change in the future.

I therefore suggest creating a separate class, for example `see-all-icon`, with the margin you want, and assigning it to the icon in addition to the one there already is: `See all<i class="fa-long-arrow-alt-right see-all-icon"></i>`

Reply:
Thank you, Sebastian! That's the solution I was looking for :)